docs/source/app-dev/app-arch.rst

```diff
@@ -148,3 +148,28 @@ To use command deduplication, you should:
 
 
 For more details on command deduplication, see the :ref:`Ledger API Services <command-submission-service-deduplication>` documentation.
+
+.. _dealing-with-time:
+
+Dealing with time
+*****************
+
+The DAML language contains a function :ref:`getTime() <daml-ref-gettime>` which returns the “current time”. The notion of time comes with a lot of problems in a distributed setting: different participants might run slightly different clocks, transactions would not be allowed to “overtake” each other during DAML interpretation, i.e., a long-running command could block all other commands, and many more.
+
+To avoid such problems, DAML provides the following concept of *ledger time*:
+
+- As part of command interpretation, each transaction is automatically assigned a *ledger time* by the participant server.
+- All calls to ``getTime()`` within a transaction return the *ledger time* assigned to that transaction.
+- *Ledger time* is reasonably close to real time. To avoid transactions being rejected because the assigned *ledger time* does not match the ledger's system time exactly, DAML Ledgers define a tolerance interval around its system time. Transactions with a *ledger time* outside this tolerance interval will be rejected.
```

Comment:
I'd clarify here that "system time" is part of the synchonization/consensus protocol of ledgers. Ie it's something that's agreed on by everyone, but isn't known at interpretation time so can't be used for `getTime`.